explainer.md

```diff
@@ -506,6 +507,14 @@ The change events API exposed to documents does not accept URLs.
 Service workers may pass any URL under their scopes into the query or
 change event API.
 
+#### Service Worker Scope 
+
+The Cookie Store API does not change the access level for Service Workers.
+
+We think that Service Workers can currently effectively access the cookies of any URL under their scope. For example, a Service Worker could respond to any top-level request with an HTML document embedding an `<iframe>` pointing to the desired URL. When responding to the request for that URL, the Service Worker can respond with an HTML document containing a `<script>` that implements a postMessage API that proxies the Service Worker's access to the `document.cookie` API.
```

Comment:
Suggest removing "We think that" and "effectively".

And "implements a postMessage API" is a bit awkward. Maybe rewrite the last bit to: that proxies the Service Worker's access to the `document.cookie` API using `postMessage`.

Reply:
:+1: